resources/js/App/Components/Layout/Layout.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./layout.css"
+
+import Header from "../Header/Header.jsx";
+import LayoutLeft from "../LayoutLeft/LayoutLeft.jsx";
+import LayoutRight from "../LayoutRight/LayoutRight.jsx";
+
+
+export default class Layout extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      desktop: true,
+      width: window.innerWidth,
+      videoPageOn: false,
+    }
+  }
+
+  componentWillMount() {
+    if(this.state.width < 900) {
+      this.setState( { desktop: false } )
+    }
+  }
+
+  isVideoPageOn = () => {
```

Comment:
This could be done much simpler:
`isVideoPageOn = () => this.setState({videoPageOn: !this.state.videoPageOn})`